### ISSUE TITLE
NVDA should announce the document information  such as title bar current page number etc

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2833]
**Reported by anubhav on 2012-11-29 11:20**
NVDA should announce the document information  such as title bar current page number etc. However, NVDA is announceing same twice

**Comment by ehollig** [comment]
I am not exactly sure what the original poster is referring to. Any ideas @Brian1Gaff?

**Comment by leonardder** [comment]
I'm closing this as invalid. 

1. I can't reproduce this in MS Word or LibreOffice Writer, the only major products I know of that report page numbers.
2. @Brian1Gaff rightly asked for more details from the author, which hasn't been provided.
